stage0/fstar-lib/FStarC_Tactics_Native.ml

```diff
@@ -0,0 +1,102 @@
+open FStarC_Compiler_Range
```

Comment:
These files (hand-written ML, F*, ulib, etc.) all show up in the git diff.  Should we mark them as binaries like the checked in ML files?